src/ui/play_mode.rs

```diff
@@ -81,6 +87,40 @@ fn data_inspector(ui: &mut egui::Ui, world: &mut World) {
     }
 
     ui.add_space(10.0);
+
+    ui.horizontal(|ui| {
+        ui.label("Method:");
+
+        let methods = ["BruteForceKNN", "KdTree", "KMeans"];
+        let mut selected_index = match selected_method.method {
+            Method::BruteForceKNN => 0,
+            Method::KdTree => 1,
+            Method::KMeans => 2,
+        };
```

Comment:
```suggestion
        let mut selected_index = _method_state.get() as usize;
```
You can cast an `enum` into a `usize`